src/style.css

```diff
@@ -1 +1,46 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
According to the checklist, you should not use the '*' selector as it impacts performance. Even though you are not using the '*' selector, applying the box-sizing to all elements with the 'body' selector can also impact the performance. It's better to apply box-sizing only to the elements that need it.